<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="51" total="52" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-09-01T19:10:54 IST" name="Default suite" finished-at="2023-09-01T19:12:10 IST" duration-ms="76267">
    <groups>
    </groups>
    <test started-at="2023-09-01T19:10:54 IST" name="Default test" finished-at="2023-09-01T19:12:10 IST" duration-ms="76267">
      <class name="Tests.Registry_Module3_MCII_Test">
        <test-method is-config="true" signature="start()[pri:0, instance:Tests.Registry_Module3_MCII_Test@73e9cf30]" started-at="2023-09-01T19:10:54 IST" name="start" finished-at="2023-09-01T19:10:54 IST" duration-ms="9" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- start -->
        <test-method is-config="true" signature="setup()[pri:0, instance:Tests.Registry_Module3_MCII_Test@73e9cf30]" started-at="2023-09-01T19:10:54 IST" name="setup" finished-at="2023-09-01T19:10:57 IST" duration-ms="3465" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Validation_of_Quantity_against_Reserve_Budget_for_Price_Containment_Reserve_Allowance_Issuance()[pri:51, instance:Tests.Registry_Module3_MCII_Test@73e9cf30]" started-at="2023-09-01T19:10:57 IST" name="Validation_of_Quantity_against_Reserve_Budget_for_Price_Containment_Reserve_Allowance_Issuance" finished-at="2023-09-01T19:12:10 IST" duration-ms="72570" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [The issuance has been successfully prepared and is awaiting review.] but found [The allowance quantity exceeds the allowances that can be issued from the Reserve Budget]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [The issuance has been successfully prepared and is awaiting review.] but found [The allowance quantity exceeds the allowances that can be issued from the Reserve Budget]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at pageObjects.Registry_Module3_MCII.validateQuantity_PCRAIssuance(Registry_Module3_MCII.java:3346)
at Tests.Registry_Module3_MCII_Test.Validation_of_Quantity_against_Reserve_Budget_for_Price_Containment_Reserve_Allowance_Issuance(Registry_Module3_MCII_Test.java:694)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Validation_of_Quantity_against_Reserve_Budget_for_Price_Containment_Reserve_Allowance_Issuance -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:Tests.Registry_Module3_MCII_Test@73e9cf30]" started-at="2023-09-01T19:12:10 IST" name="closeBrowser" finished-at="2023-09-01T19:12:10 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="addResultToTestrail(org.testng.ITestResult)[pri:0, instance:Tests.Registry_Module3_MCII_Test@73e9cf30]" started-at="2023-09-01T19:12:10 IST" name="addResultToTestrail" finished-at="2023-09-01T19:12:10 IST" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Validation_of_Quantity_against_Reserve_Budget_for_Price_Containment_Reserve_Allowance_Issuance status=FAILURE method=Registry_Module3_MCII_Test.Validation_of_Quantity_against_Reserve_Budget_for_Price_Containment_Reserve_Allowance_Issuance()[pri:51, instance:Tests.Registry_Module3_MCII_Test@73e9cf30] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addResultToTestrail -->
        <test-method is-config="true" signature="sentReportToTheMail()[pri:0, instance:Tests.Registry_Module3_MCII_Test@73e9cf30]" started-at="2023-09-01T19:12:10 IST" name="sentReportToTheMail" finished-at="2023-09-01T19:12:10 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sentReportToTheMail -->
      </class> <!-- Tests.Registry_Module3_MCII_Test -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
